boot/zephyr/Kconfig

```diff
@@ -72,37 +72,37 @@ config MBEDTLS_CFG_FILE
 	default "mcuboot-mbedtls-cfg.h"
 
 config BOOT_VALIDATE_SLOT0
```

Comment:
I would change this keyword as well. It produces `#define CONFIG_BOOT_VALIDATE_SLOT0 1` currently.

Reply:
`mcuboot_config.h` is checking for this define as is here to avoid breaking compatibility with previous custom configurations, so I don't think it needs to be changed for now.

Reply:
"Previous custom configuration" - do you mena proj.conf file in this repository or end-user configuration?

Reply:
If someone made `BOOT_VALIDATE_SLOT0=n` in a custom config

Reply:
ok - can live with this

Reply:
Or maybe we can keep backward compatibility:

```suggestion
config BOOT_VALIDATE_SLOT0
    # Hidden selector for backward compatibility
    # deprecated
    bool
    select BOOT_VALIDATE_PRIMARY_SLOT
    
config BOOT_VALIDATE_PRIMARY_SLOT
```

Reply:
Are we happy with it as it is now?

Reply:
I am

Reply:
But it's a nice suggestion from @nvlsianpu...